src/docker.ts

```diff
@@ -78,7 +78,11 @@ export async function startContainer (actionCfg: ActionConfiguration, ctxConfig:
 export async function cleanupContainers (): Promise<void> {
   await stopContainer(process.env.PIPER_ACTION_dockerContainerID ?? '')
   await stopContainer(process.env.PIPER_ACTION_sidecarContainerID ?? '')
-  await removeNetwork()
+  await removeNetwork(process.env.PIPER_ACTION_dockerNetworkID ?? '')
```

Comment:
Are you sure that this works in a parallel scenario? Wouldn't the variable overwrite eachother?